plugins/org.jkiss.dbeaver.tasks.ui/src/org/jkiss/dbeaver/tasks/ui/wizard/TaskConfigurationWizardPageTask.java

```diff
@@ -213,6 +215,37 @@ public void createControl(Composite parent) {
                     UIUtils.createLabel(typePanel, task.getType().getName());
                 }
             }
+            Composite advancedPanel = UIUtils.createControlGroup(
+                composite,
+                TaskUIMessages.task_config_wizard_page_task_advanced_label, 2,
+                GridData.FILL_HORIZONTAL, 0);
+            maxExecutionTimeBtn = UIUtils.createCheckbox(
+                advancedPanel,
+                TaskUIMessages.task_config_wizard_page_task_max_exec_time,
+                TaskUIMessages.task_config_wizard_page_task_max_exec_time_descr,
+                true,
+                1);
+            maxExecutionTime = UIUtils.createSpinner(advancedPanel, null, 1, 1, Integer.MAX_VALUE);
+            maxExecutionTimeBtn.addSelectionListener(new SelectionAdapter() {
+                @Override
+                public void widgetSelected(
+                    SelectionEvent e) {
```

Comment:
```suggestion
                public void widgetSelected(SelectionEvent e) {
```
Or rather use `org.eclipse.swt.events.SelectionListener#widgetSelectedAdapter`